tests/timeseries/mock_microgrid.py

```diff
@@ -48,16 +49,28 @@ class MockMicrogrid:  # pylint: disable=too-many-instance-attributes
     battery_id_suffix = 9
 
     _microgrid: MockMicrogridClient
-
-    def __init__(
-        self, grid_side_meter: bool, num_values: int = 2000, sample_rate_s: float = 0.01
+    mock_data: MockDataPipeline
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        grid_side_meter: bool,
+        api_client_streaming: bool = False,
+        num_values: int = 2000,
+        sample_rate_s: float = 0.01,
+        namespaces: int = 1,
```

Comment:
optionally I'd rename `namespaces` to `num_spaces` to better describe the variable and be consistent with other variables like `num_values`

Reply:
renamed to `num_namespaces`.